### PR TITLE
Fwf 1083

### PR DESCRIPTION
**Pull request by shuhaib-aot**
# Issue Tracking

JIRA: 

# Changes

# How has the change been tested?

# Screenshots (if applicable)

# Build Success screenshot (Till a CICD pipeline is set up)

# Notes

### CODE REVIEW

**Comment by kurianbenoy-aot** [comment]
@shuhaib-aot raise PR to dev

**Comment by shuhaib-aot** [comment]
PR added to dev

File: camunda-formio-tasklist-vue/src/services/CamundaRest.ts [hunk 1]
```diff
@@ -209,7 +209,7 @@ export const CamundaRest = {
```
Comment (kurianbenoy-aot) [inline camunda-formio-tasklist-vue/src/services/CamundaRest.ts:209]: ).then(async () => {
      await this.getGroupDetails();
      await this.reloadCurrentTask();
    });
    this.setGroup = null;
  }

Can this async and await be removed?

Comment (kurianbenoy-aot) [reply camunda-formio-tasklist-vue/src/services/CamundaRest.ts:209]: addGroup () {
    CamundaRest.createTaskGroupByID(
      this.token,
      this.task.id!,
      this.bpmApiUrl,
      {
        userId: null,
        groupId: this.setGroup,
        type: "candidate",
      }
    ).then(() => {
      this.getGroupDetails();
      this.reloadCurrentTask();
    });
    this.setGroup = null;
  }

Comment (shuhaib-aot) [reply camunda-formio-tasklist-vue/src/services/CamundaRest.ts:209]: removed

Comment (kurianbenoy-aot) [reply camunda-formio-tasklist-vue/src/services/CamundaRest.ts:209]: @shuhaib-aot thanks for resolving. I am pretty sure more such issue are there in codebase. When you are working on Code optimization ticket, you can work on it.

**Comment by kurianbenoy-aot** [comment]
There was a lint issue @shuhaib-aot , updated it

**Pull request merged**